modules/custom/d_commerce/d_commerce.module

```diff
@@ -20,6 +20,10 @@ function d_commerce_theme($existing, $type, $theme, $path) {
     'commerce_coupon_redemption_form' => [
       'template' => 'd_commerce-coupon-redemption-form',
     ],
+    'commerce_checkout_completion_message' => [
+      'template' => 'commerce-checkout--droopler--completion-message',
```

Comment:
tu mam jeszcze wątpliwości co do nazwy twiga,
może ustalmy jakąś wspólną konwencję. Ja tam dodaję kolejnego twiga i postanowiłem nazwać go tak jak oryginal skopiowany z commerca. 
W pozostałych nazwach są d_commerce oraz droopler - a to chyba niepotrzebne jest.

![image](https://user-images.githubusercontent.com/37533838/81992151-21aa3700-9643-11ea-8a90-a7a364a5b9a4.png)

Reply:
Hmmm ja to ugryzłem jeszcze inaczej. Tzn. własnie sie zorientowalem, ze dodałem też tempate tego typu ale do d_commerce_product (bedzie to trzeba pozniej przeniesc).
W kazdym razie mam o  taki plik: commerce-order-total-summary--default.html.twig
![image](https://user-images.githubusercontent.com/42933966/82013776-64880100-967b-11ea-94ca-bdba9dcc75bb.png)
żeby nie nadpisywać innych order typów :-) (gdyby ktoś chciał dodać coś swojego i jakoś sobie z customizować).
Z drugiej strony, całkiem możliwe, że ktoś by nadal chciał mieć zachowane te same style itp. Sam nie wiem :thinking:

Reply:
Tak jak zrobił to Mariusz jest dobrze i tak powinniśmy nazwać te twigi